### PR TITLE
removing cis 1.2.2

### PR DESCRIPTION
**Pull request by oren-zohar**
Since we moved to the CIS [kube 1.20 benchmark](https://drive.google.com/drive/u/0/folders/1UTr2xp5gtQJ-T5SUeOP18EC-tKKECymY) there were some changes, one is that 1.2.2 was removed.

@ofiriro3 the diff got crazy, but I just removed 1.2.2 and patched the subsequent rules.

### CODE REVIEW

File: compliance/cis_k8s/rules/cis_1_1_3/rule.rego [hunk 1]
```diff
@@ -4,7 +4,7 @@ import data.compliance.cis_k8s
 import data.compliance.lib.common
 import data.compliance.lib.data_adapter
 
-# Ensure that the API server pod specification file permissions are set to 644 or more restrictive
+# Ensure that the controller manager pod specification file permissions are set to 644 or more restrictive (Automated)
 finding = result {
 	data_adapter.filename == "kube-controller-manager.yaml"
 	filemode := data_adapter.filemode
```
Comment (ofiriro3) [inline compliance/cis_k8s/rules/cis_1_1_3/rule.rego:10]: Can you add `default_value` to metadata?

File: compliance/cis_k8s/rules/cis_1_2_13/rule.rego [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+	# set result
+	result := {
+		"evaluation": common.calculate_result(rule_evaluation),
+		"evidence": {"process_args": process_args},
+	}
+}
+
+metadata = {
+	"name": "Ensure that the admission control plugin ServiceAccount is set",
```
Comment (ofiriro3) [inline compliance/cis_k8s/rules/cis_1_2_13/rule.rego:32]: Description should be - `Automate service accounts management.`

File: compliance/cis_k8s/rules/cis_1_2_13/rule.rego [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	"description": "When you create a pod, if you do not specify a service account, it is automatically assigned the default service account in the same namespace. You should create your own service account and let the API server manage its security tokens.",
```
Comment (ofiriro3) [inline compliance/cis_k8s/rules/cis_1_2_13/rule.rego:33]: Just change the name "description" to "Rationale:" and add the correct description

File: compliance/cis_k8s/rules/cis_1_2_25/rule.rego [hunk 4]
```diff
@@ -4,16 +4,16 @@ import data.compliance.cis_k8s
...
 
-# Ensure that the --audit-log-maxsize argument is set to 100 or as appropriate (Automated)
+# Ensure that the --request-timeout argument is set as appropriate (Automated)
 
 # evaluate
 process_args := data_adapter.process_args
 
 default rule_evaluation = false
 
 rule_evaluation {
```
Comment (ofiriro3) [inline compliance/cis_k8s/rules/cis_1_2_25/rule.rego:14]: > But, setting this timeout limit to be too large can exhaust the API server resources making it
prone to Denial-of-Service attack.

I think that the rule needs to verify that the request timeout doesn't exceed 60 seconds.

Comment (oren-zohar) [reply compliance/cis_k8s/rules/cis_1_2_25/rule.rego:14]: let's open a ticket for anything unrelated to the issue scope you find

**Review by ofiriro3 — APPROVED** [review]
GJ - Please go over my comment to see if there is indeed a bug.

If there is one, please open a task

File: compliance/cis_k8s/rules/cis_1_2_25/test.rego [hunk 5]
```diff
@@ -5,12 +5,14 @@ import data.lib.test
 
 test_violation {
 	test.assert_fail(finding) with input as rule_input("")
-	test.assert_fail(finding) with input as rule_input("--audit-log-maxsize=99")
+	test.assert_fail(finding) with input as rule_input("--request-timeout=30s")
```
[the remark this answers is not in the document]
Comment (oren-zohar) [reply compliance/cis_k8s/rules/cis_1_2_25/test.rego:8]: not part of this PR scope

**Pull request merged**